### DISCUSSION TITLE
Skip images without digits and take a new one?

### DISCUSSION THREAD

**Discussion by fab-ge** [jomjol/AI-on-the-edge-device#2934] [General]
Hi,

thank you for this wonderful project. Works great on my water meter.

Now I would like to read my heat meter /Landis Gyr heat meter as well.
It uses an LCD display which shows no digits for 5 seconds, then 1 second the total amount consumed.

I know that the ESP cam requires several minutes to process a picture before it can take the next picture. 

Would it be possible to implement a "burst-mode" where a few pictures are taken a few seconds appart from each other, then delete all the pictures which do not show any value and process the first one which shows digits?

Thanks in advance
Fabian

**Comment by caco3** [comment]
We do not have a plan for such a feature.
There are other people with similar wishes (electricity meter which toggles between 2 tariffs.
The solution usually is to set the neg rate detection and max rate detection correctly, so an empty or partial image gets dropped.
